apps/language_server/lib/language_server/providers/execute_command/manipulate_pipes.ex

```diff
@@ -230,7 +238,11 @@ defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes do
         col + tail_length
       end
 
-    {:ok, pipe_call, range(start_line, start_col, end_line, end_col)}
+    if String.contains?(pipe_call, "|>") do
```

Comment:
@axelson I've added this because your test actually was calling "fromPipe", which shouldn't do anything if there aren't any pipes in the selected text

Reply:
Another possible option which would require some rework is to scan until a pipe is found, but this could have unexpected results